charts/collectorset-controller/templates/collector-rbac.yaml

```diff
@@ -41,6 +41,7 @@ rules:
       - endpoints
       - pods/log
       - events
+      - configmaps
```

Comment:
If this is added for configmap monitoring, I guess this is not required.

Reply:
This is required to pull the configMap data while running configSource.